codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectEncoder.java

```diff
@@ -187,6 +197,10 @@ private void encodeChunkedContent(ChannelHandlerContext ctx, Object msg, long co
         }
     }
 
+    boolean isContentAlwaysEmpty(@SuppressWarnings("unused") H msg) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "msg". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)